spec/end_to_end/cfn_nag/input_template_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe 'cfn_nag template file input', end_to_end: true do
+  context 'when reading in a template file with 1 failure' do
+    it "should print out 'failure_count': 1 " do
```

Comment:
nit: i'd scrub "should" from the it clauses.  like yoda would say, do - or do not, there is no try.  e.g. it "prints out failure count 1"